Sources/Rego/IREvaluator.swift

```diff
@@ -314,11 +426,11 @@ internal struct Frame {
 // Scopes are children of a Frame, representing a logical Rego scope. Each time
 // a statement has nested blocks a new Scope is pushed. These are primarily used
 // for further protecting locals and tracking "with" value overrides.
-internal struct Scope: Encodable {
```

Comment:
Does the `Locals()` change make this no longer Encodable? What are the implications, is this acceptable? 🤔